### PR TITLE
feat: ticker add platform

### PR DESCRIPTION
**Pull request by jphuc96**
**What does this PR do?**

-   [x] My bugfix

**How to test**

-   Create a detailed description of what you need to do to set this PR up. ie: Does it need migrations? Do you need to install something?
-   Create a step by step list of what the engineer needs to do to test.

**Edge cases**

-   What are the edge cases that you might see in this PR?

**Media (Loom or gif)**

-   Insert media here (if applicable)

### CODE REVIEW

File: src/commands/ticker/index/processor.ts [hunk 1]
```diff
@@ -203,6 +203,14 @@ export async function composeTickerResponse({
       value: current,
       inline: true,
     },
+    {
+      name: "Platform",
+      value:
+        coin.asset_platform_id && coin.asset_platform_id !== ""
+          ? coin.asset_platform_id.toUpperCase()
+          : "N/A",
+      inline: true,
```
Comment (tuanddd) [inline src/commands/ticker/index/processor.ts:212]:
```suggestion
      value: (coin.asset_platform_id || "N/A").toUpperCase(),
      inline: true,
```

**Review by tuanddd — APPROVED** [review]

File: src/commands/ticker/index/processor.ts (hunk 1 above)
Comment (jphuc96) [reply src/commands/ticker/index/processor.ts:212]: can exclude "" nua ser
